src/GraphQL.Harness.Tests/GraphQL.Harness.Tests.csproj

```diff
@@ -2,11 +2,11 @@
   <Import Project="../Tests.props" />
 
   <PropertyGroup Condition="'$(SingleTestPlatform)' == 'true'">
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFrameworks>$(SingleFramework)</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(SingleTestPlatform)' != 'true'">
-    <TargetFrameworks>net7.0;net6.0;net5.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net7.0;net6.0;net5.0</TargetFrameworks>
   </PropertyGroup>
```

Comment:
Why these are still in the test project? These tests can't run on `net48`?

Reply:
No, GraphQL.Harness is designed for ASP.NET Core 3.x and above.  ASP.NET Core 2.x (which supports netfx) is notably different, with the libraries resting in nuget packages instead of within the shared framework.  I'm sure it could be made compatible with extra effort, although the testing library Alba currently used by GraphQL.Harness.Tests does not support ASP.NET Core 2.x

Reply:
Not compatible packages. I see.

Reply:
You may be interested in reviewing the GraphQL.Server csproj and how we made that work for both netfx and .NET Core:

https://github.com/graphql-dotnet/server/blob/master/src/Transports.AspNetCore/Transports.AspNetCore.csproj

Note that although it may be possible to run ASP.NET Core 2.x on .NET Core 3.x, that scenario is not supported by the Server project.  It also does not target .NET 3.0, which means it's probably not even possible to use GraphQL.NET Server on .NET 3.0 at all.

There are also separate sample projects there for netfx vs .NET Core, which is where the end-to-end tests reside.